src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1294,7 +1294,7 @@ internal static bool PathIsUnc(string path)
             }
 
             // handle special cases like \\wsl$\ubuntu and \\?\ which aren't a UNC path, but we can say it is so the filesystemprovider can use it
-            if (path.StartsWith(WslRootPath, StringComparison.OrdinalIgnoreCase) || path.StartsWith("\\\\?\\"))
+            if (!networkOnly && (path.StartsWith(WslRootPath, StringComparison.OrdinalIgnoreCase) || path.StartsWith("\\\\?\\")))
```

Comment:
Makes sense to add `\\.\` too?

Reply:
I don't believe you can use the period to refer to the local computer as part of a UNC path, it has to be the computer name, localhost, 127.0.0.1, etc...

Reply:
What about @fMichaleczek's example `Get-ChildItem '\\.\pipe\'`?

Reply:
That doesn't work for me even in WinPS 5.1

Reply:
This works for me in WinPS 5.1 but not in cmd.exe.
![image](https://user-images.githubusercontent.com/22290914/129134889-e94c48cd-6456-436c-9a60-a6e299874015.png)

Reply:
/cc @jborean93

Reply:
@iSazonov

```
\\ServerName\pipe\PipeName

where ServerName is either the name of a remote computer or a period, to specify the local computer.

```
The reference page : 
https://docs.microsoft.com/en-us/windows/win32/ipc/pipe-names

Reply:
Still not working for me, do I need to do something else?

![image](https://user-images.githubusercontent.com/11859881/129416935-7a2e8196-f97d-428f-b04a-b409e8ec90bf.png)

Reply:
Ok, looks like the trailing backslash is REQUIRED

Reply:
I'll include `\\.\` and add a test